app/src/main/res/layout/fragment_home.xml

```diff
@@ -48,5 +53,11 @@
             android:layout_marginTop="8dp"
             android:textSize="14sp"
             />
+    <View
```

Comment:
You dont need even creating an extra view for performance sake. You can just create a background color on recylerView and a slightly different color on the item view then set the item view to have a margin bottom. This will do

Reply:
okay i did what you said, i think i have finished the issue, can i close it?